src/views/tutorial-view/index.tsx

```diff
@@ -1,13 +1,23 @@
+import Content from '@hashicorp/react-content'
```

Comment:
💭 I didn't know this existed! I think we need to get one of these for DevDot... we're basically doing the same thing in `SidebarSidecarLayout` for applying special styles.

For more context, some of the components we're passing into `components` are: `Heading`, `InlineLink`, and `Text`. You can see how they're used for docs MDX here: https://github.com/hashicorp/dev-portal/blob/assembly-ui-v1/src/layouts/sidebar-sidecar/utils/_local_platform-docs-mdx/index.jsx.

Reply:
Oh interesting! Y'all aren't using the react-content component? Is there a different component I should use to get the dev dot styles?